magicctapipe/scripts/lst1_magic/semi_automatic_scripts/coincident_events.py

```diff
@@ -62,7 +62,7 @@ def configfile_coincidence(target_dir, source_name, config_gen):
         config_dict = yaml.safe_load(fc)
 
     conf = {
```

Comment:
with both the mc_tel_ids and event_coincidence taken from base_config_file you are basically using directly this base_config_file. So in principle we could dump the whole "config_file" into file_name (or even copy a file) to simplify the code, but having the config file stripped of all unnecessary settings also has its advantages, so perfectly fine for me to keep it like this as well.

Reply:
Indeed the idea was to split the (quite huge) config.yaml file in a set of subfiles (one per step), each one containing only the information needed in this step (both user-friendly and, since we have to open this configuration file in all the jobs to read it, maybe also a bit more 'SLURM-friendly')

Reply:
ok, perfectly fine with me